src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/ptv/TestPtvConsts.java

```diff
@@ -0,0 +1,75 @@
+package fi.otavanopisto.kuntaapi.test.server.integration.ptv;
+
+public class TestPtvConsts {
+  
+  public final static String[] ORGANIZATIONS = {
+    "ae2682d3-6238-4019-b34f-b078c5f9bb50",
+    "f3cb50b9-e97f-4c00-a9c7-04265bdcadb6",
+    "9355a207-efd3-4cfb-a02b-67187f34c822"
+  };
+
+  public final static String[] SERVICES = {
+    "2f21448e-e461-4ad0-a87a-47bcb08e578e",
+    "bcfaf5f1-98db-401e-8628-7d4f2b4ec260",
+    "4c490d84-1fc2-4011-98b5-fbc7a878cbce",
+    "b21dbc0e-ea54-449d-8419-3fe7a6126948",
+    "6b979930-048e-4ed4-a80c-0e11ba562dfe",
+    "58a33fee-d7c6-4875-a1b7-75d70b00d012",
+    "08ab5b97-d04d-4d16-a3f3-dac654b140ec",
+    "2fb268a5-8fa5-467e-b908-98131b916365",
+    "8f0fe030-794e-4ed5-beb7-57c6be43f2ba"
+  };
+
+  public final static String[] SERVICE_CHANNELS = {
+    "b8e980f4-58c7-477b-a9d1-b01d50be50f0",
+    "0452df6a-f322-47de-af5b-767694f120cd",
+    "faa9c7b7-5b1f-4964-b04f-410558b66647",
+    "047b91ef-aa08-4be6-b72d-1231e61c5cb9",
+    "97c01787-b57c-42f1-a002-7d09b363993f",
+    "74b3ac81-cf57-40b0-b0b1-b9dcf752a143",
+    "b5fba182-2a08-44aa-a168-c11f8d1d4664",
+    "614bdc8a-bb55-4f36-be4e-7e1bea9a7706",
+    "dda27a67-1b20-4966-a57e-162ca4c54ef5",
+    "4d52960e-2e39-4d9e-8f7c-5bc8994b024f",
+    "0c00262c-3529-4cc3-b0fe-2f03a2c4e4ce",
+    "1dec7c89-96b8-4860-83d0-bad8b8897ba6",
+    "a31fbcc7-d45f-4eb2-a32a-afd134d02aa5",
+    "b0ca2d94-f113-47ae-9c90-1bc1733c44be",
+    "1af0ec8c-b647-4f40-b1a9-db6052c5096d",
+    "155b896e-f1f5-43db-9497-aa3a8afd2487",
+    "0e4abaca-f57e-4649-a42d-0f421d09e9fd"
+  };
+
+  public final static String[] ELECTRONIC_CHANNEL_SERVICE_CHANNELS = {
+    "b8e980f4-58c7-477b-a9d1-b01d50be50f0",
+    "614bdc8a-bb55-4f36-be4e-7e1bea9a7706"
+  };
+
+  public final static String[] SERVICE_LOCATION_SERVICE_CHANNELS = {
+    "0452df6a-f322-47de-af5b-767694f120cd",
+    "faa9c7b7-5b1f-4964-b04f-410558b66647",
+    "97c01787-b57c-42f1-a002-7d09b363993f",
+    "b5fba182-2a08-44aa-a168-c11f8d1d4664",
+    "dda27a67-1b20-4966-a57e-162ca4c54ef5",
+    "4d52960e-2e39-4d9e-8f7c-5bc8994b024f",
+    "1dec7c89-96b8-4860-83d0-bad8b8897ba6",
+    "a31fbcc7-d45f-4eb2-a32a-afd134d02aa5",
+    "155b896e-f1f5-43db-9497-aa3a8afd2487",
+    "0e4abaca-f57e-4649-a42d-0f421d09e9fd"
+  };
+
+  public final static String[] PRINTABLE_FORM_SERVICE_CHANNELS = {
+    "047b91ef-aa08-4be6-b72d-1231e61c5cb9",
+    "1af0ec8c-b647-4f40-b1a9-db6052c5096d"
+  };
+
+  public final static String[] PHONE_SERVICE_CHANNELS = {
+    "0c00262c-3529-4cc3-b0fe-2f03a2c4e4ce",
+    "b0ca2d94-f113-47ae-9c90-1bc1733c44be"
+  };
+
+  public final static String[] WEB_PAGE_SERVICE_CHANNELS = {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this member "protected". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2386)